### DISCUSSION TITLE
targets trouble - parcel2

### DISCUSSION THREAD

**Discussion by sabr49** [parcel-bundler/parcel#5359] [Q&A]
Thanks for the awesome project. Hoping you could help me - I am trying to create a multi-build project. I have a simple package.config file:

![image](https://user-images.githubusercontent.com/1566251/99446223-bb9db580-28ec-11eb-8b0b-f86848f71508.png)

I also have a babelrc file with the following:

![image](https://user-images.githubusercontent.com/1566251/99446307-d6702a00-28ec-11eb-9dc0-b2582b0b23c6.png)

When I run ```npx parcel build  --target old --no-source-maps    src/index.js```

I would expect the output from @babel/preset-env to show in the Using Targets section that we are targetting ie 11 (my understanding is that the browsers is passed along as the build environment). Instead I see this:

![image](https://user-images.githubusercontent.com/1566251/99446891-0f100380-28ed-11eb-84a7-f0b7ad34a035.png)

This is an example of a build with a browserslist set to ie 11 -

![image](https://user-images.githubusercontent.com/1566251/99447577-41216580-28ed-11eb-941f-1f012806e3fa.png)

Any guidance is appreciated. I happen to be using ```"parcel": "^2.0.0-nightly.447"``` but its not specific to that - when I use @next I have the same results.

**Comment by mischnic** [comment]
You have to use `@parcel/babel-preset-env` instead so that these target specific engines are forwarded properly. This preset behaves just liek `@babel/preset-env` apart from that (options, etc...).

**Reply by sabr49** [reply]
Thanks so much for the quick reply! That was it.
